backend/src/main/java/com/ambacam/transfert/operateurs/OperateurSearchTO.java

```diff
@@ -0,0 +1,19 @@
+package com.ambacam.transfert.operateurs;
+
+import java.util.List;
+
+import com.ambacam.transfert.SearchResultTO;
+
+public class OperateurSearchTO extends SearchResultTO {
+
+	private List<OperateurReadTO> operateurs;
```

Comment:
In the swagger this should be a normal operateur

Reply:
The swagger has been now updated.  But now the description of the models and those of the entpoint diverge. What should be considered for get? 
If we should consider for the models please ignore this comment and update the swagger.

Reply:
Yes, we should consider the models.
Sorry for that. I update the Swagger right now